### PR TITLE
add serviceable to nuget packages by default

### PR DESCRIPTION
**Pull request by rohit21agrawal**
Fixes : https://github.com/NuGet/Home/issues/2952#issuecomment-225684719

@rrelyea @emgarten @alpaix @joelverhagen

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ curl -o cli/dotnet-install.sh https://raw.githubusercontent.com/dotnet/cli/rel/1
 
 # Run install.sh
 chmod +x cli/dotnet-install.sh
-cli/dotnet-install.sh -i cli -c preview -v 1.0.0-preview2-002911
+cli/dotnet-install.sh -i cli -c preview -v 1.0.0-preview2-003030
```
Comment (joelverhagen) [inline build.sh:20]: presumable this version of `dotnet pack` allows the `--servicable` option?

**Comment by rrelyea** [comment]
:lgtm:

File: build.sh (hunk 1 above)
Comment (rohit21agrawal) [inline build.sh:20]: yes.

Comment (rrelyea) [inline build.sh:20]: yes, that is what we believe.

**Comment by joelverhagen** [comment]
:shipit:

**Comment by rohit21agrawal** [comment]
3.5.0-beta2 : https://github.com/NuGet/NuGet.Client/commit/b26a26ed7e812cd607cf7ba0a62ab7e2c3ca2b1f

File: build/common.ps1 [hunk 2]
```diff
@@ -383,7 +383,7 @@ Function Invoke-DotnetPack {
             if($ReleaseLabel -ne 'Release') {
                 $opts += '--version-suffix', "${ReleaseLabel}-${BuildNumber}"
             }
-
+            $opts += '--serviceable'
```
Comment (alpaix) [inline build/common.ps1:386]: This would actually build all the packages with serviceable attribute including test ones, but I think that's fine

**Pull request merged**